live-examples/css-examples/pseudo-class/meta.json

```
@@ -18,6 +18,24 @@
             "defaultTab": "css",
             "height": "tabbed-shorter"
         },
+        "disabled": {
+            "cssExampleSrc": "./live-examples/css-examples/pseudo-class/disabled.css",
+            "exampleCode": "./live-examples/css-examples/pseudo-class/disabled.html",
+            "fileName": "pseudo-class-disabled.html",
+            "title": "CSS Demo: :disabled",
+            "type": "tabbed",
+            "defaultTab": "css",
+            "height": "tabbed-shorter"
```

```suggestion
            "height": "tabbed-standard"
```

(the HTML overflows)